### ISSUE TITLE
Add dark and light theme 

### ISSUE THREAD

**Issue by goyal-Dushi** [goyal-Dushi/my-projects#1]
Use **Material UI** icons and `makeStyles` to add styles to components. Don't add any extra stylesheet like `CSS` or `SCSS` files.
Place the toggle icon in the Homenavbar component, having the Appbar.

**Comment by Shashanks0616** [comment]
u have to add icons at the navbar portion like {home , about us, contact us}

**Comment by goyal-Dushi** [comment]
> u have to add icons at the navbar portion like {home , about us, contact us}

No, just the theme toggler icon, like Moon for dark theme, Sun for the light theme. Or you could use Material [Switch](https://material-ui.com/components/switches/).
